package.json

```
@@ -38,7 +38,7 @@
     "@iden3/bigarray": "0.0.2",
     "@iden3/binfileutils": "0.0.6",
     "fastfile": "0.0.19",
```

I'd go a step further and use semver on all these so they can get the latest updates
```suggestion
    "@iden3/bigarray": "^0.0.2",
    "@iden3/binfileutils": "^0.0.6",
    "fastfile": "^0.0.19",
```